packages/integration-tests/tests/dashboard.test.ts

```diff
@@ -50,31 +32,20 @@ describe('admin console dashboard', () => {
   });
 
   it('should get active user statistics successfully', async () => {
-    type ActiveUserStatistics = {
-      dauCurve: StatisticsData[];
-      dau: StatisticsData;
-      wau: StatisticsData;
-      mau: StatisticsData;
-    };
-
-    const originActiveUserStatistics = await authedAdminApi
-      .get('dashboard/users/active')
-      .json<ActiveUserStatistics>();
+    const originActiveUserStatistics = await getActiveUsersData();
 
-    await registerUserAndSignIn();
+    await signIn(username, password);
```

Comment:
This test relies on the test above (register user); per discussion on [slack](https://silverhand-io.slack.com/archives/C02A8G4HVAM/p1658315031191559), should we keep them independent?

Reply:
should be good after making the whole test process linear. Anyway updated.